src/ui/features/onboarding/Onboarding.tsx

```diff
@@ -45,6 +46,14 @@ export function Onboarding() {
           </PageLayout>
         }
       />
+      <Route
+        path="/onboarding/backup/*"
+        element={
+          <PageLayout>
+            <Backup />
+          </PageLayout>
```

Comment:
https://github.com/zeriontech/zerion-wallet-extension/pull/569/files#r1680870245

Though if you also wrap in `<PageLayout />` here, I don't see why it can't just be inside `<Backup />`?

Reply:
I see, it's done here for consistency with sibling routes